src/application/controller/controller.h

```diff
@@ -0,0 +1,83 @@
+#ifndef CONTROLLER_H_
+#define CONTROLLER_H_
+
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "FreeRTOS.h"
+#include "state_estimation.h"
+#include "flight_phase.h"
+#include "queue.h"
+#include "math.h"
+#include "common.h"
+
+
+
+/* Enums/Types */
+
+
+// input from state estimation module
+typedef struct {
+    quaternion_t attitude; // Current attitude vector
+    vector3d_t rates; // Current angular rates
+    vector3d_t velocity; // Current velocity vector
+    float altitude; // Current altitude
+    float timestamp; // Timestamp in ms
+    float canard_coeff_CL; // Canard coefficient
+    float canard_angle_delta; // Canard angle
+} controller_input_t;
+
+
+// main controller state using in task
+typedef struct {
+    controller_input_t current_state;
```

Comment:
Design doc says this should be of type `controller_state_t` but that isn't defined elsewhere. I'm assuming thats out of date and should be `controller_input_t ` in the doc?